README.md

```diff
@@ -1,109 +1,88 @@
-# Luxtorpeda-dev
+# 🚋 Luxtorpeda: Enhancing your gaming experience on Linux and Steam Deck
```

Comment:
I think this should stay as being called Luxtorpeda Dev, to avoid any confusion.